spec/person_spec.rb

```diff
@@ -0,0 +1,19 @@
+require_relative '../book'
+require_relative '../persons'
+
+describe Person do
+  context 'containing a list of names and ages' do
+    person = Person.new(45, 'Margaret')
+
+    it 'expects Margaret age to be 45' do
+      expect(person.age).to eq 45
+    end
+    it 'expects Person.name to be Margaret' do
+      expect(person.name).to eq 'Margaret'
+    end
+
+    it 'expect can_use_services to equal true' do
+      expect(person.can_use_services?).to eq true
+    end
```

Comment:
- [ ] When you create a person than a rental array is created :thinking: You didn't check that Kindly test your rentals array using the following code :arrow_down:

`expect(person.rentals).to eql []`